fab/inventory/production

```diff
@@ -60,5 +62,5 @@ datavol_device=/dev/xvdb
 [stanchion]
 hqriak0.internal-va.commcarehq.org
 
-[rackspace_cloud_backup]
-hqdb0.internal.commcarehq.org
+# [rackspace_cloud_backup]
+# hqdb0.internal-va.commcarehq.org
```

Comment:
@esoergel I think we saw this was deprecated when you were setting up new VA machines correct? https://github.com/dimagi/commcarehq-ansible/pull/452